lib/application/widgets/theme/themed_bottom_container.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
```

Comment:
This file name could be just `themed_container` instead `themed_bottom_container`, since it has both: `top` and `bottom`